src/Receiving.js

```diff
@@ -3,15 +3,28 @@ import {
   Switch,
   Route,
 } from 'react-router-dom';
+import ReactRouterPropTypes from 'react-router-prop-types';
 
 import { ReceivingListContainer } from './ReceivingList';
+import { TitleFormContainer } from './TitleForm';
 
-const Receiving = () => {
+const Receiving = ({ match }) => {
   return (
     <Switch>
-      <Route component={ReceivingListContainer} />
+      <Route
+        component={TitleFormContainer}
+        path="/receiving/create"
+      />
+      <Route
+        component={ReceivingListContainer}
+        path="/receiving"
+      />
     </Switch>
   );
 };
 
+Receiving.propTypes = {
+  match: ReactRouterPropTypes.match.isRequired,
```

Comment:
is it used?

Reply:
for now - no, but we will require this at the Edit route